server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -4,6 +4,7 @@
  */
 
 import { assert } from "console";
+import { cloneDeep } from "lodash";
```

Comment:
Nit (which doesn't matter as much for server as for client anyway): starting with Node17 [`structuredClone()`](https://nodejs.org/docs/latest-v18.x/api/globals.html#structuredclonevalue-options) is available so might be able to avoid the extra dep.

Reply:
@alexvy86  I saw the `structuredClone` but just feel too early to use it. But anyway, I changed to use it, and fallback to other older approaches if we failed.